src/config/fair-queue.php

```diff
@@ -1,10 +0,0 @@
-<?php
```

Comment:
properties inside this file will be merged with the database configuration config file. please don't delete this.

oh BTW what happened to that piece of code inside service provider?!

`$this->mergeConfigFrom(
            __DIR__ . '/config/fair-queue.php',
            'database.redis.fair-queue'
        );`